src/routes/Favourites.js

```diff
@@ -166,32 +170,36 @@ function Favourites() {
           }
           return (
             <>
-              {data.subDomainFavourites.map(domain => (
-                <Query
-                  query={GET_OWNER}
-                  variables={{ name: domain.name }}
-                  key={domain.name}
-                >
-                  {({ loading, error, data }) => {
-                    if (error)
+              {data &&
+                data.subDomainFavourites &&
+                data.subDomainFavourites.map(domain => (
```

Comment:
optional chaining

Reply:
Same. optional chaining is not supported on IE and opera.